docs/guide/provider-migration.md

```diff
@@ -58,11 +53,11 @@ We recommend [`ethers`](https://npmjs.com/package/ethers) ([documentation](https
 
 ::: warning
 We strongly recommend that you consider one of the other two migration paths before resorting to this one.
-It is not future-proof, and it is not guaranteed to work.
+It is not future-proof, we will not add new features to it.
 :::
 
 Finally, if you just want your web3 site to continue to work, we created [`@metamask/legacy-web3`](https://npmjs.com/package/@metamask/legacy-web3).
-This package is a drop-in replacement for our `window.web3` that you can add to your web3 site even before MetaMask stops injecting `window.web3`.
+This package is a drop-in replacement for our `window.web3` that you can add to your website even before we ship the `window.web3` on all platforms.
```

Comment:
I think you accidentally a word here :thinking:

Reply:
Addressed in: 1065b9e